charts/kamaji/templates/etcd_rbac.yaml

```diff
@@ -15,7 +15,7 @@ rules:
     resources:
       - secrets
     verbs:
-      - delete
```

Comment:
The `delete` is still required for the post-delete job of the `etcd` component.

We should rather add the `get` verb and not remove the `delete` one.

Reply:
Yep, I missed that. Has been re-added in the latest commit